### ISSUE TITLE
Render stop signs

### ISSUE THREAD

**Issue by wmisener** [ZeLonewolf/openstreetmap-americana#796]
Per discussion related to traffic signals https://github.com/ZeLonewolf/openstreetmap-americana/issues/795#issuecomment-1434905662, I'm opening a new issue to track rendering stop signs (`highway=stop`).

Stop signs are a ubiquitous feature of low-traffic American roads, moreso than in many other countries, such as in Europe. This perhaps partly explains their lack of rendering in OSM projects like Carto, where rendering them [was declined](https://github.com/gravitystorm/openstreetmap-carto/issues/3366) by the maintainers.

I think it would be worthwhile for this style to render these features once you're zoomed in, maybe at z>17 or so. I think their locations are useful for people on a variety of modes at a neighborhood/street level. Also, Americana is perhaps more of a "road map" than some other renderers, so their appearance would be more appropriate. On a personal level, I like adding stop signs to my local area, and so I'd appreciate the mapper feedback that other renderers like Carto currently lack for these common features ([more than 1.3 million mapped worldwide](https://taginfo.openstreetmap.org/tags/highway=stop)). The obvious icon would be an octagon, which could be red, though a black octagon would probably also be sufficiently recognizable and might fit in with the existing white fill on the road renderings better.

Stop signs are not currently in OMT, so they would need to be added to the schema before we could render them.

**Comment by 1ec5** [comment]
I agree that stop signs would be as relevant to the style as traffic signals. Unfortunately, the only examples of stop signs I can find in print maps are in tourist maps that aim for a cartoonish take on a realistic representation of traffic control features like stop signs and traffic signals. Even PSMA Australia, publisher of the incredibly detailed Sydways/Melways/etc. maps that meld American and British cartography, doesn’t seem to show stop signs. (There should be one at the end of [Sherwood Street](http://www.street-directory.com.au/?x=144.99128092459327&y=-37.819838310002055&l=14).)
